src/RatingSystem.js

```diff
@@ -2,9 +2,7 @@ import React, { useState } from 'react'
 import styled from 'styled-components/macro'
 import { FaSquare } from 'react-icons/fa'
 
-export default function RatingSystem() {
-  const [rating, setRating] = useState(null)
-
```

Comment:
du kannst in Zeile 1 den useState rausnehmen, da der Hook gelöscht wurde

Reply:
oh ja :)